akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/Dialect.scala

```diff
@@ -413,11 +418,15 @@ private[projection] case class MSSQLServerDialect(
  * INTERNAL API
  */
 @InternalApi
-private[projection] case class OracleDialect(schema: Option[String], tableName: String, managementTableName: String)
+private[projection] case class OracleDialect(schema: Option[String], _tableName: String, _managementTableName: String)
     extends Dialect {
 
   def this(tableName: String, managementTableName: String) = this(None, tableName, managementTableName)
 
+  override val tableName: String = _tableName.toUpperCase(Locale.ROOT)
```

Comment:
@ihostage Thanks for fixing. Could you give us some background on the problem? This must always have been a problem? Why didn't tests fail?

Reply:
> Could you give us some background on the problem?

Yes, of course. How I see this story:
1. SQL scripts for Oracle in docs and default table name in [reference.conf (version 1.0.0)](https://github.com/akka/akka-projection/blob/v1.0.0/akka-projection-jdbc/src/main/resources/reference.conf) were UPPER case format.
2. Then Renato @octonato fixed other problems in https://github.com/akka/akka-projection/pull/499
3. Now SQL scripts for Oracle in docs are UPPER case format, but the default table name in [reference.conf (version 1.1.0)](https://github.com/akka/akka-projection/blob/v1.1.0/akka-projection-jdbc/src/main/resources/reference.conf) is lower case format.
4. I created all tables by docs and for version 1.1.0 `JdbcOffsetStore` throws `ORA-00942: table or view does not exist`.
5. If I sets `akka.projection.jdbc.offset-store.[table|management-table]` manually by UPPER format then all are fine.
6. But I think that docs should be sync with default settings and users don't need to do something for the base case.

Reply:
Thanks for explaining. Makes sense.

Reply:
@ihostage Can you look into the test failure in OracleJdbcOffsetStoreSpec?

Reply:
Yes, I fixed already and test locally. Waiting for what will say GA.

Reply:
Ooops... Flaky https://github.com/akka/akka-projection/issues/544